src/cdk/explorer/nodes/propertyNode.ts

```diff
@@ -0,0 +1,52 @@
+/*!
+ * Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { ext } from '../../../shared/extensionGlobals'
+import { AWSTreeNodeBase } from '../../../shared/treeview/nodes/awsTreeNodeBase'
+
+/*
+ * Represents a property of a CDK construct. Properties can be simple key-value pairs, Arrays,
+ * or objects that are nested deeply
+ */
+export class PropertyNode extends AWSTreeNodeBase {
+    public constructor(
+        label: string,
+        collapsibleState: vscode.TreeItemCollapsibleState,
+        private readonly subProperties?: { [key: string]: any }
```

Comment:
Assuming subProperties is of the `ConstructProps` type?

Reply:
see above ^^ ConstructProps has a key called props that contains the properties/values in there. 
It will contain more than just those props though